src/main/proto/ga4gh/rna_quantification.proto

```diff
@@ -34,31 +34,47 @@ message RnaQuantification {
   // Description
   string description = 3;
 
-  // ID of the ReadGroup providing the reads for the analysis.
-  string read_group_id = 4;
+  // ID(s) of the ReadGroup(s) providing the reads for the analysis.
+  repeated string read_group_ids = 4;
 
-  // List of programIds used in the analysis.
-  repeated string program_ids = 5;
+  // Program can be used to track the provenance of how read data was quantified.
+  message Program {
```

Comment:
Support this suggestion. ```assaymetadata``` would be the right place (see structure in https://github.com/ga4gh/schemas/pull/606).